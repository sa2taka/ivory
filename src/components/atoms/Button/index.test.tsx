import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Button } from './index';

describe('Button', () => {
  test('is rendered', () => {
    const wrapper = shallow(<Button>button</Button>);
    expect(wrapper).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
