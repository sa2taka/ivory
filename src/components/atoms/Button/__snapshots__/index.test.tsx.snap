// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button is rendered 1`] = `
<_Button
  theme={Object {}}
>
  button
</_Button>
`;
